docs/source/changelog.rst

```diff
@@ -8,6 +8,26 @@ All notable changes to this project will be documented in this file.
 `Unreleased`_
 -------------
 
+Added
+^^^^^
```

Comment:
A readme, is for developpers using the library. Don't right anything about testing inside (TestStateBackend, freezegun)